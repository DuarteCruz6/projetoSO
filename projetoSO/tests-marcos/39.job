WRITE [(s3,39)(y4,43)(a0,69)(v8,71)]
WRITE [(i0,31)(j4,66)(p1,17)(p0,24)(w8,51)]
WRITE [(m5,77)(r5,76)(a4,84)(g4,33)]
WRITE [(j7,47)(f4,37)(k9,16)]
DELETE [j7,a0]
WRITE [(t5,11)(b9,52)(t0,80)(i6,18)]
READ [t5]
READ [t5]
WRITE [(g8,88)(o4,35)(v4,33)(u5,99)(s6,94)]
READ [s3]
WRITE [(b2,21)(u0,51)]
DELETE [u5,j4]
WRITE [(w2,29)(x9,27)]
SHOW
READ [a4]
WAIT 1000
WRITE [(k2,34)]
SHOW
READ [x9]
WRITE [(i7,13)]
WRITE [(y6,49)(p4,40)(h1,35)]
READ [t0]
DELETE [t0,u0,m5]
DELETE [y4,h1,t5]
WRITE [(b3,91)(u3,77)(t2,32)]
BACKUP
SHOW
WRITE [(a0,79)(n6,10)(x3,6)(u7,8)(m7,0)]
READ [p1]
DELETE [r5]
READ [i6]
READ [w8]
DELETE [s6,i6]
WRITE [(l7,49)(x6,28)]
WRITE [(c4,56)(o6,63)(k4,35)(h8,33)]
WRITE [(t0,95)]
WRITE [(s6,59)(p3,63)(l4,38)(v5,39)(v7,52)]
WRITE [(a1,50)(m4,42)(u9,4)]
DELETE [x9,i0,b2,w2]
