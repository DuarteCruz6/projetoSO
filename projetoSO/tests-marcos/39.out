[(t5,11)]
[(t5,11)]
[(s3,39)]
(a4, 84)
(b2, 21)
(b9, 52)
(f4, 37)
(g8, 88)
(g4, 33)
(i6, 18)
(i0, 31)
(k9, 16)
(m5, 77)
(o4, 35)
(p0, 24)
(p1, 17)
(r5, 76)
(s6, 94)
(s3, 39)
(t0, 80)
(t5, 11)
(u0, 51)
(v4, 33)
(v8, 71)
(w2, 29)
(w8, 51)
(x9, 27)
(y4, 43)
[(a4,84)]
(a4, 84)
(b2, 21)
(b9, 52)
(f4, 37)
(g8, 88)
(g4, 33)
(i6, 18)
(i0, 31)
(k2, 34)
(k9, 16)
(m5, 77)
(o4, 35)
(p0, 24)
(p1, 17)
(r5, 76)
(s6, 94)
(s3, 39)
(t0, 80)
(t5, 11)
(u0, 51)
(v4, 33)
(v8, 71)
(w2, 29)
(w8, 51)
(x9, 27)
(y4, 43)
[(x9,27)]
[(t0,80)]
(a4, 84)
(b3, 91)
(b2, 21)
(b9, 52)
(f4, 37)
(g8, 88)
(g4, 33)
(i7, 13)
(i6, 18)
(i0, 31)
(k2, 34)
(k9, 16)
(o4, 35)
(p4, 40)
(p0, 24)
(p1, 17)
(r5, 76)
(s6, 94)
(s3, 39)
(t2, 32)
(u3, 77)
(v4, 33)
(v8, 71)
(w2, 29)
(w8, 51)
(x9, 27)
(y6, 49)
[(p1,17)]
[(i6,18)]
[(w8,51)]
