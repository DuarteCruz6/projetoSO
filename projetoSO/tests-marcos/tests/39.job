BACKUP
WRITE [(f8,59)(f6,95)(x5,78)(s5,82)(a8,79)]
WRITE [(p9,1)(r7,35)(n3,89)(q7,95)]
WRITE [(d0,62)(s0,0)]
WRITE [(t6,22)]
WRITE [(t5,54)(k7,34)(a5,57)]
WRITE [(j6,7)(n8,16)]
WRITE [(p6,96)]
WRITE [(t4,62)]
DELETE [p6,f6,t6,n3,t4]
WRITE [(e9,44)(s9,34)(d3,23)(g8,28)]
DELETE [s0]
WRITE [(e1,75)(u6,79)]
READ [a8]
DELETE [s5,q7,x5,a8]
WRITE [(g1,17)(y6,39)(s2,42)(o7,85)]
WRITE [(b5,35)(w6,5)(i3,0)]
READ [p9]
WRITE [(b4,85)(r0,13)]
DELETE [e9,s9,u6]
DELETE [g1]
WRITE [(c9,93)]
READ [a5]
WRITE [(o8,83)(s6,79)(k5,38)(t7,19)]
BACKUP
READ [s2]
READ [f8]
READ [o8]
READ [n8]
WRITE [(x7,98)(p2,98)(i4,87)(m2,12)]
DELETE [w6,i4]
READ [b5]
WRITE [(h8,52)(y3,55)(e0,35)(o5,12)(f1,85)]
WRITE [(b3,73)(l7,42)]
WRITE [(h4,67)]
WRITE [(v4,94)(h2,15)(s9,45)(x6,13)]
DELETE [j6,l7,r0]
BACKUP
WRITE [(l9,2)(d6,63)(u3,12)(y7,83)(v6,95)]
READ [d3]
READ [f1]
WRITE [(u8,100)(m9,73)(i8,5)(c8,84)(f9,56)]
DELETE [c8,r7,e0,s9]
WRITE [(l8,39)(e9,49)(z7,50)(q2,38)(w3,59)]
READ [y6]
WRITE [(a6,79)(q3,24)(n6,2)(q6,60)(y4,12)]
READ [b5]
WAIT 1000
WRITE [(k3,34)(d8,83)(e4,68)(h3,73)]
BACKUP
READ [y7]
DELETE [p9,o8]
WAIT 1000
WRITE [(v0,84)(c3,21)(c2,55)(y2,1)]
WAIT 1000
WRITE [(g7,38)(q9,77)(c0,49)(h9,59)]
READ [b5]
BACKUP
WRITE [(v3,83)(t3,16)(n5,38)(m7,63)(c4,41)]
BACKUP
WRITE [(g5,1)(r7,30)]
WRITE [(r3,30)(w4,45)(z6,6)(p9,9)]
READ [n6]
WRITE [(l5,72)(j1,7)(g6,77)(g6,78)]
READ [b4]
WRITE [(r8,78)(j5,18)(d5,72)(k4,39)]
READ [g7]
SHOW
WRITE [(t2,1)(u7,20)(j0,92)(n7,95)]
READ [h8]
WRITE [(e3,53)(x1,44)]
WRITE [(d9,36)(q8,44)(x0,71)(g1,43)(a7,84)]
WRITE [(j3,92)]
WRITE [(g9,59)(o9,29)(l4,92)(y8,87)(b0,48)]
WRITE [(s7,41)]
WRITE [(j2,39)(f3,89)(i5,92)]
READ [r3]
WRITE [(r6,58)(n2,17)(z3,11)]
WRITE [(c8,4)]
DELETE [y3,d9,p9,x0,x1]
WRITE [(o3,52)]
WRITE [(h6,37)(t4,1)(w0,97)]
WRITE [(u2,11)(t0,53)]
DELETE [w3,j1,t0]
WRITE [(s8,21)(w1,86)]
WRITE [(o0,80)(p3,57)]
