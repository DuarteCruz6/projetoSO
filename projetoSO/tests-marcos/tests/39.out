[(a8,79)]
[(p9,1)]
[(a5,57)]
[(s2,42)]
[(f8,59)]
[(o8,83)]
[(n8,16)]
[(b5,35)]
[(d3,23)]
[(f1,85)]
[(y6,39)]
[(b5,35)]
[(y7,83)]
[(b5,35)]
[(n6,2)]
[(b4,85)]
[(g7,38)]
(a6, 79)
(a5, 57)
(b3, 73)
(b4, 85)
(b5, 35)
(c4, 41)
(c0, 49)
(c2, 55)
(c3, 21)
(c9, 93)
(d5, 72)
(d8, 83)
(d6, 63)
(d3, 23)
(d0, 62)
(e4, 68)
(e9, 49)
(e1, 75)
(f9, 56)
(f1, 85)
(f8, 59)
(g6, 78)
(g5, 1)
(g7, 38)
(g8, 28)
(h9, 59)
(h3, 73)
(h2, 15)
(h4, 67)
(h8, 52)
(i8, 5)
(i3, 0)
(j5, 18)
(j1, 7)
(k4, 39)
(k3, 34)
(k5, 38)
(k7, 34)
(l5, 72)
(l8, 39)
(l9, 2)
(m7, 63)
(m9, 73)
(m2, 12)
(n5, 38)
(n6, 2)
(n8, 16)
(o5, 12)
(o7, 85)
(p9, 9)
(p2, 98)
(q9, 77)
(q6, 60)
(q3, 24)
(q2, 38)
(r8, 78)
(r3, 30)
(r7, 30)
(s6, 79)
(s2, 42)
(t3, 16)
(t7, 19)
(t5, 54)
(u8, 100)
(u3, 12)
(v3, 83)
(v0, 84)
(v6, 95)
(v4, 94)
(w4, 45)
(w3, 59)
(x6, 13)
(x7, 98)
(y2, 1)
(y4, 12)
(y7, 83)
(y3, 55)
(y6, 39)
(z6, 6)
(z7, 50)
[(h8,52)]
[(r3,30)]
